Blog.API/Blog.API/Middlewares/GlobalExceptionHandlingMiddleware.cs

```diff
@@ -16,21 +23,19 @@ public async Task InvokeAsync(HttpContext context, RequestDelegate next)
         }
         catch (Exception ex)
         {
+            _logger.LogError($"An unhandled exception occured. Exception {typeof(Exception)}", ex);
+
```

Comment:
We need to know lots of other information, not only ex. The best way I know is ex.ToString(). So it should be like this: 
```suggestion
            _logger.LogError("An unhandled exception occurred. Exception: {error}", ex.ToString());
```